notebooks/explainable-ai-1-basic/explainable-ai-1-basic.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    %pip install -q  "openvino-xai==1.0.0rc1"
</pre><p>What is release schedule for openvino-xai? Generally, we avoid strict versions for installing packages for reduce maintenance overhead (when you need to update version for every new release), is it possible to use "openvino-xai&gt;=1.0.0"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Sure, updated